compat/src/suspense-list.js

```diff
@@ -23,7 +23,7 @@ const resolve = (list, child, node) => {
 		// mark the child as completely resolved by deleting it from ._map.
 		// This is used to figure out when *all* children have been completely
 		// resolved when revealOrder is 'together'.
-		list._map.delete(child);
+		list._map.delete(child._node);
```

Comment:
Something is wrong here, since we'll only delete one and the second one isn't present in the _map (wrong ref?)